src/bttrx_fsm.cpp

```diff
@@ -114,7 +116,13 @@ void BTTRX_FSM::handleStateConfigure()
 	}
 
 	// Enforce configuration
-	string friendly_name = "bt-trx-" + wt32i_.getBDAddressSuffix();
+	string friendly_name = "bt-trx_";
+	String callsign = preferences.getString("callsign", "");
+	if (callsign != "") {
+		friendly_name = friendly_name + callsign.c_str();
+	} else {
+		friendly_name = friendly_name + wt32i_.getBDAddressSuffix();
+	}
```

Comment:
OK, although you should probably use `!callsign.empty()` for checking against an empty string.

Reply:
'''
src/bttrx_fsm.cpp: In member function 'void BTTRX_FSM::handleStateConfigure()':
src/bttrx_fsm.cpp:121:16: error: 'class String' has no member named 'empty'
  if (!callsign.empty()) {
                ^
'''

Need to dig further into this.

Reply:
Done with 888408e3974973b4779f6f34db16dabc96b020a2